pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -201,6 +201,27 @@ func (o *GinkgoRunSuiteOptions) Run(suite *TestSuite, junitSuiteName string, mon
 		logrus.Infof("Using built-in tests only due to OPENSHIFT_SKIP_EXTERNAL_TESTS being set")
 	}
 
+	// Temporarily check for the presence of the [Skipped:xyz] annotation in the test names, once this synthetic test
+	// begins to pass we can remove the annotation logic
+	var annotatedSkipped []string
+	for _, t := range externalTestCases {
+		if strings.Contains(t.name, "[Skipped") {
+			annotatedSkipped = append(annotatedSkipped, t.name)
+		}
+	}
+	var skippedAnnotationSyntheticTestResults []*junitapi.JUnitTestCase
+	skippedAnnotationSyntheticTestResult := junitapi.JUnitTestCase{
+		Name: "Skipped annotations present",
+	}
+	if len(annotatedSkipped) > 0 {
+		skippedAnnotationSyntheticTestResult.FailureOutput = &junitapi.FailureOutput{
+			Message: fmt.Sprintf("Skipped Annotations present in tests: %s", strings.Join(annotatedSkipped, ", ")),
+		}
+	}
+	skippedAnnotationSyntheticTestResults = append(skippedAnnotationSyntheticTestResults, &skippedAnnotationSyntheticTestResult)
+	// Make it flake
+	skippedAnnotationSyntheticTestResults = append(skippedAnnotationSyntheticTestResults, &junitapi.JUnitTestCase{Name: skippedAnnotationSyntheticTestResult.Name})
```

Comment:
Should we only flake when we have a failure?

Reply:
I guess it would only flake if there was a failure, right? It would just be 2 successful results otherwise. I could update the comment to make that more clear.

Reply:
Yeah, I'm not sure if the flake logic looks for more than one run with at least one pass or more than one run with one failure.  We could ask, it just seemed odd to me to add the second pass in the case it wasn't needed.

Reply:
Just to keep the code simple, but I get what you are saying. I wouldn't imagine that 2 successful runs would result in a flake, but maybe my assumption is wrong.

Either way, this is intended to be pretty temporary, so I don't think we need to debate the topic too much ;) If you want, I will wrap it in an if statement.